PWGHF/D2H/Tasks/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ o2physics_add_dpl_workflow(task-dstar-to-d0-pi
 
 o2physics_add_dpl_workflow(task-flow-charm-hadrons
                     SOURCES taskFlowCharmHadrons.cxx
-                    PUBLIC_LINK_LIBRARIES O2Physics::AnalysisCore
+                    PUBLIC_LINK_LIBRARIES O2Physics::AnalysisCore O2Physics::EventFilteringUtils
```

Comment:
Why do you need to add this library?

Reply:
Hi @vkucera ! Thanks for the review. I need it to include the HF utility function for event selection, otherwise, the build fails

Reply:
Ah, I see. OK